testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-ldap-and-file/password-authenticator.properties

```diff
@@ -0,0 +1,7 @@
+password-authenticator.name=ldap
```

Comment:
Where does this file come from? Looks like it did not exit previously?

Reply:
Previously it was referred from a common class now we have duplicated it specific to this class.